var/spack/repos/builtin/packages/chombo/package.py

```diff
@@ -0,0 +1,130 @@
+##############################################################################
+# Copyright (c) 2013-2016, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/llnl/spack
+# Please also see the LICENSE file for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+from shutil import copyfile
+import glob
+
+
+class Chombo(MakefilePackage):
+    """The Chombo package provides a set of tools for implementing finite
+       difference and finite-volume methods for the solution of partial
+       differential equations on block-structured adaptively refined
+       logically rectangular (i.e. Cartesian) grids. Note: Chombo download
+       requires human registration process."""
+
+    homepage = "https://anag-repo.lbl.gov/svn/Chombo/release"
+    url      = "https://anag-repo.lbl.gov/svn/Chombo/release/3.2"
+
+    version('3.2', svn='https://anag-repo.lbl.gov/svn/Chombo/release/3.2')
```

Comment:
Ok, I've had a telephone call with Brain Van Straalen regarding Chomb in Spack. He will be sending me a github path, the same one I think 3133 is currently using and it'll become a github download rather than a less save, unchecksumed, svn download. Satisfactory?